allennlp/data/dataset_readers/sequence_tagging.py

```diff
@@ -2,7 +2,7 @@
 
 from overrides import overrides
 
-from allennlp.data.dataset_readers import DatasetReader
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
```

Comment:
Why is this a problem for `DatasetReader` and not for `Params`, which is also imported from the module?